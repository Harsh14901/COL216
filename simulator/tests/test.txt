sw $s2, 2048($zero)
sw $s2, 2048($zero)
sw $s2, 2048($zero)
