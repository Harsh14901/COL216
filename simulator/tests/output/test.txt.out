[+] Compiling file : tests/input/test.txt
[+] Program compiled successfully !
[+] Executing program ...
[+] Executing in NON-BLOCKING MODE
---------- Execution logs ----------
[$] CYCLE 1-1 :
[#] Executing current instruction -> sw $s2, 2048($zero)
[#] Register contents -
[#] Row buffer updates:
[#] Memory updates:
[#] Remarks: 
	DRAM request issued
============================================================
[$] CYCLE 2-2 :
[#] Executing current instruction -> sw $s2, 2048($zero)
[#] Register contents -
[#] Row buffer updates:
[#] Memory updates:
[#] Remarks: 
	DRAM request issued
============================================================
[$] CYCLE 3-3 :
[#] Executing current instruction -> sw $s2, 2048($zero)
[#] Register contents -
[#] Row buffer updates:
[#] Memory updates:
[#] Remarks: 
	DRAM request issued
============================================================
[$] CYCLE 2-11 :
[#] Executing current instruction -> 
[#] Register contents -
[#] Row buffer updates:
[#] Memory updates:
[#] Remarks: 
	DRAM: activating row 2
============================================================
[$] CYCLE 12-13 :
[#] Executing current instruction -> 
[#] Register contents -
[#] Row buffer updates:
	2048-2051 : 0
[#] Memory updates:
[#] Remarks: 
	DRAM: Writing 0 to row buffer @ 2048
============================================================
[$] CYCLE 14-15 :
[#] Executing current instruction -> 
[#] Register contents -
	 $zero :          0		   $s0 :          0
	   $at :          0		   $s1 :          0
	   $v0 :          0		   $s2 :          0
	   $v1 :          0		   $s3 :          0
	   $a0 :          0		   $s4 :          0
	   $a1 :          0		   $s5 :          0
	   $a2 :          0		   $s6 :          0
	   $a3 :          0		   $s7 :          0
	   $t0 :          0		   $t8 :          0
	   $t1 :          0		   $t9 :          0
	   $t2 :          0		   $k0 :          0
	   $t3 :          0		   $k1 :          0
	   $t4 :          0		   $gp :          0
	   $t5 :          0		   $sp :    1048576
	   $t6 :          0		   $fp :          0
	   $t7 :          0		   $ra :          0
[#] Row buffer updates:
	2048-2051 : 0
[#] Memory updates:
[#] Remarks: 
	DRAM: Writing 0 to row buffer @ 2048
============================================================
---------- Execution statistics ----------
[#] Clock cycles: 15
[#] Number of row buffer updates: 3
[#] Updated memory : 
	2048-2051 : 0
[+] Program terminated
